.circleci/config.yml

```diff
@@ -1,17 +1,27 @@
 version: 2.1
-orbs:
-  node: circleci/node@1.1.6
 jobs:
-  build-and-test:
-    executor:
-      name: node/default
+  build:
+    docker:
+      - image: circleci/node:10
```

Comment:
10/12/14?

Reply:
Will do a fast follow once i figure out how to get circle-ci to do multi nodes. Otherwise I think we should look to move to github actions